commands/inventory/add_agent_qan_mongodb_profiler_agent.go

```diff
@@ -100,4 +118,7 @@ func init() {
 	AddAgentQANMongoDBProfilerAgentC.Flag("skip-connection-check", "Skip connection check").BoolVar(&AddAgentQANMongoDBProfilerAgent.SkipConnectionCheck)
 	AddAgentQANMongoDBProfilerAgentC.Flag("tls", "Use TLS to connect to the database").BoolVar(&AddAgentQANMongoDBProfilerAgent.TLS)
 	AddAgentQANMongoDBProfilerAgentC.Flag("tls-skip-verify", "Skip TLS certificates validation").BoolVar(&AddAgentQANMongoDBProfilerAgent.TLSSkipVerify)
+	AddAgentQANMongoDBProfilerAgentC.Flag("tls-certificate-key-file", "Define certificate").StringVar(&AddAgentQANMongoDBProfilerAgent.TLSCertificateKeyFile)
```

Comment:
The word `define` is not clear.
Can we just say "TLS certtificate pem file"?

Reply:
Yes, it is better.